util/runtests.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+# Can be executed with the options:
+#   --unit-test for the standard extended unit-testing report (default when no option is provided)
+#   --pytest for the shortened pytest summary report
```

Comment:
Can you add a comment stating whether it is it also possible to run it with both options, or not.

Reply:
Done